### DISCUSSION TITLE
want a highlight feature

### DISCUSSION THREAD

**Discussion by bigfishtwo** [fury-gl/fury#522] [Ideas / New Features]
Hi, I find that your visuliazation of streamlines is really nice, but I want to highlight one or several streamlines to analyse how the curve looks like, I find that I can only change the color, instead of hightlight,  I want the effect of glowing lines. 
Hope this is a useful suggestion.

**Comment by skoudoro** [comment]
Hi @bigfishtwo,

Thank you for the feedback. For sure we do not have this feature done on FURY but you can easily add it or play with the opacity level. Look [here](https://github.com/dipy/dipy/blob/master/dipy/viz/app.py#L19). They have done it with `dipy_horizon`.
I hope it helps. I move this issue to a discussion
